examples/misc_controls_arcball.html

```diff
@@ -0,0 +1,268 @@
+					folderOptions.add( controls, 'reset' ).name( 'Reset' );
+					folderAnimations.add( controls, 'enableAnimations' ).name( 'Enable anim.' );
+					folderAnimations.add( controls, 'dampingFactor', 0, 100, 1 ).name( 'Damping' );
+					folderAnimations.add( controls, 'wMax', 0, 100, 1 ).name( 'Angular spd' );
+
+				}
+
+			};
+
+
+			init();
+
+			function init() {
+
+				const container = document.createElement( 'div' );
+				document.body.appendChild( container );
+
+				renderer = new THREE.WebGLRenderer( { antialias: true, alpha: true } );
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				
+				renderer.outputEncoding = THREE.sRGBEncoding;
+				renderer.toneMapping = THREE.ReinhardToneMapping;
+				renderer.toneMappingExposure = 3;
+				renderer.domElement.style.background = "linear-gradient( 180deg, rgba( 0,0,0,1 ) 0%, rgba( 128,128,255,1 ) 100% )";
+				container.appendChild( renderer.domElement );
+
+				//
+
+				scene = new THREE.Scene();
+				scene.add( new THREE.HemisphereLight( 0x443333, 0x222233, 4 ) );
+
+				camera = makePerspectiveCamera();
+				camera.position.set( 0, 0, perspectiveDistance );
+
+				const material = new THREE.MeshStandardMaterial();
+
+				new OBJLoader()
+					.setPath( 'models/obj/cerberus/' )
+					.load( 'Cerberus.obj', function ( group ) {
+
+						const textureLoader = new THREE.TextureLoader().setPath( 'models/obj/cerberus/' );
+						
+						material.roughness = 1;
+						material.metalness = 1;
+
+						const diffuseMap = textureLoader.load( 'Cerberus_A.jpg', render );
+						diffuseMap.encoding = THREE.sRGBEncoding;
+						material.map = diffuseMap;
+
+						material.metalnessMap = material.roughnessMap = textureLoader.load( 'Cerberus_RM.jpg', render);
+						material.normalMap = textureLoader.load( 'Cerberus_N.jpg', render );
+
+						material.map.wrapS = THREE.RepeatWrapping;
+						material.roughnessMap.wrapS = THREE.RepeatWrapping;
+						material.metalnessMap.wrapS = THREE.RepeatWrapping;
+						material.normalMap.wrapS = THREE.RepeatWrapping;
+
+
+						group.traverse( function ( child ) {
+
+							if ( child.isMesh ) {
+							
+								child.material = material;
+							
+							}
+
+						} );
+
+						group.rotation.y = Math.PI / 2;
+						group.position.x += 0.25;
+						scene.add( group );
+						render();
+
+						new RGBELoader()
+							.setDataType( THREE.UnsignedByteType )
+							.setPath( 'textures/equirectangular/' )
+							.load( 'venice_sunset_1k.hdr', function ( hdrEquirect ) {
+
+								const pmremGenerator = new THREE.PMREMGenerator( renderer );
+								pmremGenerator.compileEquirectangularShader();
+
+								const hdrCubeRenderTarget = pmremGenerator.fromEquirectangular( hdrEquirect );
+								hdrEquirect.dispose();
+
+								//scene.background = hdrCubeRenderTarget.texture;
+								scene.environment = hdrCubeRenderTarget.texture;
```

Comment:
No need to work with `PMREMGenerator` anymore. This component is now integrated in the renderer. Just do this:
```js
hdrEquirect.mapping = THREE.EquirectangularReflectionMapping;

scene.environment = hdrEquirect;